can_tools/scrapers/official/PA/philadelhpia_vaccine.py

```diff
@@ -1,140 +1,54 @@
-                "race",
-                "ethnicity",
-            ],
-        )
-        return out
-
-    def _normalize_age(self, df):
-        df.age = df.age.str.replace("+", "_plus")
-        df.age = df.age.str.replace("<", "0-")
+    source_name = "Philadelphia Department of Public Health"
 
-        df["location_id"] = self.county_fips
-        df = self._rename_or_add_date_and_location(
-            df, date_column="etl_timestamp", location_column="location_id"
-        )
+    url = "https://healthviz.phila.gov/t/PublicHealth/views/COVIDVaccineDashboard/COVID_Vaccine?:embed=y&:showVizHome=no&:host_url=https%3A%2F%2Fhealthviz.phila.gov%2F&:embed_code_version=3&:tabs=no&:toolbar=no&:alerts=no&:showShareOptions=false&:showAskData=false&:showAppBanner=false&:isGuestRedirectFromVizportal=y&:display_spinner=no&:loadOrderID=0"
+    sheet_name = "Residents Percentage"
 
-        df = df.melt(id_vars=["location", "dt", "age"])
-        out = self.extract_CMU(
-            df,
-            self.variables,
-            [
-                "category",
-                "measurement",
-                "unit",
-                "sex",
-                "race",
-                "ethnicity",
-            ],
-        )
-        return out
-
-    def _normalize_race(self, df):
-        df = df.rename(columns={"racial_identity": "race"})
-        df.race = df.race.replace(
-            {
-                "African American": "black",
-                "Asian": "asian",
-                "Other": "other",
-                "White": "white",
-                "Unknown": "unknown",
-                "Hispanic": "hispanic",
-            }
-        )
+    variables = {
+        "fully vaccinated residents": variables.FULLY_VACCINATED_ALL,
+        "residents recieving at least 1 dose": variables.INITIATING_VACCINATIONS_ALL,
+    }
 
-        df["location_id"] = self.county_fips
-        df = self._rename_or_add_date_and_location(
-            df, date_column="etl_timestamp", location_column="location_id"
-        )
-        df = df.melt(id_vars=["location", "dt", "race"])
-        out = self.extract_CMU(
-            df,
-            self.variables,
-            [
-                "category",
-                "measurement",
-                "unit",
-                "sex",
-                "age",
-                "ethnicity",
-            ],
+    def fetch(self) -> TableauWorkbook:
+        ts = TS()
+        ts.loads(self.url)
+        return ts.getWorkbook()
+
+    def normalize(self, data: TableauWorkbook) -> pd.DataFrame:
+        sheet = data.getWorksheet(self.sheet_name)
+        df = sheet.data
+        df["Measure Names-alias"] = df["Measure Names-alias"].str.lower().str.strip()
+        found = {}
+        for target in self.variables.keys():
+            bools = df["Measure Names-alias"] == target
+            assert bools.sum() == 1
+            found[target] = int(
+                df.loc[bools, "Measure Values-alias"].iloc[0].replace(",", "")
+            )
```

Comment:
[non-blocking] Is there a reason this sort of logic is unique to the Philly dashboard, or could it be baked into `TableauWorkbook` or another helper somewhere?